src/components/SegmentedControl/src/SegmentedControl.vue

```diff
@@ -31,14 +30,6 @@ export default {
 			type: undefined,
 			required: true,
 		},
-		/**
-		 * Shape of Control & Segments
-		 */
-		shape: {
```

Comment:
Removing the shape props on components will constitute a breaking change. While our largest consuming product won't have these options available as product controls, we could leave these props alone as other consumers could benefit from them.

But even if we are set on removing them, which is reasonable, we'll need to plan for a major release and document those changes accordingly.